### PR TITLE
split_targets/test get_all_targets

### PR DESCRIPTION
**Pull request by goneri**
- add test coverage for get_all_targets() and get_targets_to_run().
- enfore the black formatter on roles/ansible-test-splitter

### CODE REVIEW

**Comment by goneri** [comment]
^ @gravesm @tremble

**Review by gravesm — APPROVED** [review]

File: roles/ansible-test-splitter/files/test_split_targets.py [hunk 1]
```diff
@@ -34,6 +58,16 @@ def test_is_slow():
...
+@patch.object(PosixPath, "glob", MagicMock(return_value=target_dir))
+def test_get_all_targets():
+    assert get_all_targets("somewhere") == {
+        "t1": ["cloud/aws", "slow"],
+        "t2": ["t1"],
+        "t3": ["cloud/aws", "", "disabled"],
+    }
+
+
+
```
Comment (tremble) [inline roles/ansible-test-splitter/files/test_split_targets.py:70]:
```suggestion
```

**Review by tremble — APPROVED** [review]
Minor lint issue (if you care), but otherwise LGTM

File: roles/ansible-test-splitter/files/test_split_targets.py (hunk 1 above)
Comment (goneri) [reply roles/ansible-test-splitter/files/test_split_targets.py]: I extended patch to fix the problem and run the black formatter through tox (and the CI).

**Pull request merged**